CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## 3.3.0 (2015-08-20)
+
+* (HParker / Adam Hess) add support for or-assign constants
+* (troessner) Update TooManyMethods default value to 15
```

Comment:
> But when I, as a reek user, upgrade reek to the next patch version I certainly do NOT expect my CI build to break because we changed smell configurations

Good point. I agree and we should update our versioning policy accordingly.